site/docs/guides/sprites.md

```diff
@@ -72,6 +72,19 @@ var smiley = [8]byte {
 }
 ```
 
+```odin
+var smiley = [8]u8 {
+    0b11000011,
+    0b10000001,
+    0b00100100,
+    0b00100100,
+    0b00000000,
+    0b00100100,
+    0b10011001,
+    0b11000011,
+}
+```
```

Comment:
Not valid Odin code.

```odin
smiley := [8]u8{...}
```

Reply:
I added a template for Odin to png2src. The template would produce this:
```odin
smiley : [8]u8 : { ... }
```
is that fine? (it compiles at least)